entity-framework/core/what-is-new/ef-core-10.0/whatsnew.md

```diff
@@ -21,6 +21,68 @@ EF10 requires the .NET 10 SDK to build and requires the .NET 10 runtime to run.
 
 ## Azure Cosmos DB for NoSQL
 
+<a name="full-text-search-support"></a>
+
+### Full-text search support
+
+Azure Cosmos DB now offers support for [full-text search](/azure/cosmos-db/gen-ai/full-text-search). It enables efficient and effective text searches, as well as evaluating the relevance of documents to a given search query. It can be used in combination with vector search to improve the accuracy of responses in some AI scenarios.
+EF Core 10 is adding support for this feature allowing for modeling the database with full-text search enabled properties and using full-text search functions inside queries targeting Azure Cosmos DB.
+
+Here is a basic EF model configuration enabling full-text search on one of the properties:
+
+```c#
+public class Blog
+{
+    ...
+
+    public string Contents { get; set; }
+}
+
+public class BloggingContext
+{
+    ...
+
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
+    {
+        modelBuilder.Entity<Blog>(b =>
+        {
+            b.Property(x => x.Contents).EnableFullTextSearch();
+            b.HasIndex(x => x.Contents).IsFullTextIndex();
+        });
+    }
+}
+```
+
+Once the model is configured, we can use full-text search operations in queries using methods provided in `EF.Functions`:
+
+```c#
+var cosmosBlogs = await context.Blogs.Where(x => EF.Functions.FullTextContains(x.Contents, "cosmos")).ToListAsync();
+```
+
+The following full-text operations are currently supported: [`FullTextContains`](/azure/cosmos-db/nosql/query/fulltextcontains), [`FullTextContainsAll`](/azure/cosmos-db/nosql/query/fulltextcontainsall), [`FullTextContainsAny`](/azure/cosmos-db/nosql/query/fulltextcontainsany), [`FullTextScore`](/azure/cosmos-db/nosql/query/fulltextscore).
+
```

Comment:
Add "For more information on Cosmos full-text search, [see the docs](...)" (IMHO this is what every section in the what's new docs should look like).

Reply:
It was already there, after Hybrid Search - hybrid was a subsection of FTS. I've elevated it to it's own section (since it's such an important concept for AI) and added the docs note for it and for FTS in general.